inst/rmarkdown/resources/css/customize-bs4.css

```
@@ -0,0 +1,96 @@
+@media screen {
+  .container-fluid {
+    padding-right: 0;
+    padding-left: 0;
+  }
+  h1:before {
+    display: none;
+  }
+}
+
+
+.book-part{
+  color: #93bcbc;
+  font-weight: bold;
+  text-transform: uppercase;
+}
+
+.li a {
+  color: #cf581b;
+}
+
+/* Ecraser les réglages de bs4 */
+@media print {
+  @page {
+    size: 210mm 297mm;
+  }
+  html, body {
+    line-height: 1.3;
+    widows: 2;
+    orphans: 2;
+  }
+  pre {
+    background-image: none;
+    border: none;
+  }
+  /* Masquer des choses */
+  button[id="print"],
+  div[data-id="main-nav"],
+  button[aria-label="Copy to clipboard"] {
+    display: none !important;
+  }
+  /* Mise en forme du titre de première page */
+
+  @page:nth(3) {
+    @top-left{
+      visibility: visible !important;
+      background-image: url(https://www.utilitr.org/images/logo-utilitr.png);
+      background-size: 2.5cm;
+      background-position: 0cm 1cm;
+      background-repeat: no-repeat;
+    }
+    @top-right{
+      width: 8cm;
+      color: black;
+      font-size: 120%;
+      font-weight: 700;
+      content: var(--date-fiche);
+      padding-top: 2cm;
+      text-align: left;
+      text-align-last: left;
+      display: block;
+    }
+  }
+  footer[class="bg-primary text-light mt-5"] {
+    display: none !important;
+  }
+
+.col-md-9 {
+  flex: 0 0 100%;
+  -webkit-flex: 0 0 100%;
+  max-width: 100%;
+  }
+  .col-xl, .col-xl-auto, .col-xl-12, .col-xl-11, .col-xl-10, .col-xl-9, .col-xl-8, .col-xl-7, .col-xl-6, .col-xl-5, .col-xl-4, .col-xl-3, .col-xl-2, .col-xl-1, .col-lg, .col-lg-auto, .col-lg-12, .col-lg-11, .col-lg-10, .col-lg-9, .col-lg-8, .col-lg-7, .col-lg-6, .col-lg-5, .col-lg-4, .col-lg-3, .col-lg-2, .col-lg-1, .col-md, .col-md-auto, .col-md-12, .col-md-11, .col-md-10, .col-md-9, .col-md-8, .col-md-7, .col-md-6, .col-md-5, .col-md-4, .col-md-3, .col-md-2, .col-md-1, .col-sm, .col-sm-auto, .col-sm-12, .col-sm-11, .col-sm-10, .col-sm-9, .col-sm-8, .col-sm-7, .col-sm-6, .col-sm-5, .col-sm-4, .col-sm-3, .col-sm-2, .col-sm-1, .col, .col-auto, .col-12, .col-11, .col-10, .col-9, .col-8, .col-7, .col-6, .col-5, .col-4, .col-3, .col-2, .col-1 {
+    padding-left: none;
+    padding-right: none;
+  }
+  /* Enlever la couleur de fond du code */
+  pre, code {
+    background-color: white !important;
+    border-color: white !important;
+    margin-left: 0 !important;
+    margin: 1em 0;
+  }
+  /* Renuméroter les pages */
+  h1 {
+    counter-reset: page 1;
+  }
+  /* Enlever les deux premières pages */
+  .pagedjs_page[data-page-number="1"],
+  .pagedjs_page[data-page-number="2"],
+  nav,
+  div[class="chapter-nav"] {
```

Pourquoi pas ça ?

```suggestion
  div.chapter-nav {
```

 ce n'est pas équivalent mais sinon ça signifie que tu as un sélecteur hyper tricky

---

Faut que je lise la doc sur les classes en CSS.